README-es.md

```diff
@@ -506,6 +506,32 @@ Estos son puntos relevantes *únicamente* para OS X.
 
 - Para obtener la información de la versión del OS X, usa `sw_vers`.
 
+## Solo para Windows
+
+Estos son puntos relevantes *únicamente* para Windows.
+
+- En Windows 10, puedes usar [Bash de Ubuntu en Windows](https://msdn.microsoft.com/commandline/wsl/about) que proporciona un entorno familiarizado con la línea de comandos en Unix. Por el lado bueno, esto permite a los programas de Linux correr en Windows y por otra parte, no permite a los programas de Windows ejecutarse en la linea de comandos Bash.
```

Comment:
**Change:**
Por el lado bueno, esto permite a los programas de Linux correr en Windows y por otra parte, no permite a los programas de Windows ejecutarse en la linea de comandos Bash.
**For:**
En el lado positivo, este permite permite a los programas de Linux ejecutarse en Windows. Por otro lado este no soporta ejecutar de programas de Windows desde la linea de comandos Bash.